client/components/ui/topbar/QuotaPreviewer.vue

```diff
@@ -0,0 +1,31 @@
+<script setup lang="ts">
+export interface QuotaPreviewerProps {
+  quota: number
+  max: number
+  unlimited?: boolean
+}
+
+const { quota, max, unlimited = computed(() => false) } = definePropsRefs<QuotaPreviewerProps>()
+const { locale, t } = useI18n()
+
+const quotaFormat = computed(() => humanSize(quota.value, locale.value, t))
+const maxFormat = computed(() => humanSize(max.value, locale.value, t))
+</script>
+
+<template>
+  <div class="flex flex-col justify-center items-center gap-1.5 h-full quota">
```

Comment:
```suggestion
  <div class="flex-center flex-col gap-1.5 h-full quota">
```

there's already a helper